web/src/components/LoginEmailCheck.tsx

```diff
@@ -0,0 +1,97 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { postUserEmailCheck } from "@/lib/api-client/apiClient";
+import { triggerSignIn } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { validateEmail } from "@/lib/utils/helpers";
+import { InputLabel } from "@mui/material";
+
+import { useState, type ReactElement } from "react";
+
+export const LoginEmailCheck = (): ReactElement => {
+  const [email, setEmail] = useState<string>("");
+  const [emailError, setEmailError] = useState<string>("");
+
+  const handleSubmit = (email: string): void => {
+    const isValidEmail = validateEmail(email);
+
+    if (isValidEmail) {
+      checkEmailExists(email);
+    } else {
+      setEmailError(Config.checkAccountEmailPage.invalidEmailError);
+    }
+  };
+
+  const checkEmailExists = async (email: string): Promise<void> => {
+    try {
+      const response = await postUserEmailCheck(email);
+      if (response.found) {
+        setEmailError("");
+        triggerSignIn();
+        analytics.event.check_account_next_button.submit.go_to_myNJ_login();
+      }
+    } catch (error) {
+      if (error === 404) {
+        setEmailError(Config.checkAccountEmailPage.emailNotFoundError);
+      }
+      if (error === 500) {
+        setEmailError(Config.checkAccountEmailPage.serviceNotAvailableError);
+      }
+    }
+  };
+
+  const { Config } = useConfig();
+
+  const handleTextInputChange = (value: string): void => {
+    setEmail(value);
+  };
+
+  return (
+    <div className="email-check-card padding-5 desktop:margin-x-7 margin-x-2 radius-md">
+      <Heading level={2} className="text-normal">
+        {Config.checkAccountEmailPage.header}
+      </Heading>
+
+      <WithErrorBar hasError={Boolean(emailError)} type="ALWAYS">
+        <InputLabel htmlFor="email">Email</InputLabel>
+        <GenericTextField
+          inputWidth="default"
+          fieldName="email"
+          value={email}
+          handleChange={handleTextInputChange}
+          error={Boolean(emailError)}
+          validationText={emailError}
+        ></GenericTextField>
+      </WithErrorBar>
+
+      <PrimaryButton
+        isFullWidthOnDesktop
+        isColor="primary"
+        isSubmitButton
+        onClick={() => handleSubmit(email)}
+      >
+        {Config.checkAccountEmailPage.inputButton}
+      </PrimaryButton>
+
+      <hr className="margin-y-3" />
+      <Content className="link-account-text">{Config.checkAccountEmailPage.linkAccountText}</Content>
+
+      <div className="display-flex flex-align-end">
+        <p>{Config.checkAccountEmailPage.needHelpText}</p>
+        <UnStyledButton
+          isUnderline
+          isIntercomEnabled
+          onClick={analytics.event.check_account_help_button.click.open_live_chat}
+          className="margin-left-05"
+        >
+          {Config.checkAccountEmailPage.intercomChatText}
```

Comment:
[dust] This will need to be tested once deployed to the dev site, unless you've tested it already in the testing environment.

Reply:
I haven't, still waiting on values for `web/src/lib/utils/analytics.ts` first